### ISSUE TITLE
Unable to install shellcheck in Ubuntu 16.04

### ISSUE THREAD

**Issue by gogasca** [koalaman/shellcheck#2092]
#### For bugs
- Rule Id (if any, e.g. SC1000): 
- My shellcheck version (`shellcheck --version` or "online"): 
- [ ] The rule's wiki page does not already cover this (e.g. https://shellcheck.net/wiki/SC2086)
- [ ] I tried on https://www.shellcheck.net/ and verified that this is still a problem on the latest commit

#### For new checks and feature suggestions
- [ ] https://www.shellcheck.net/ (i.e. the latest commit) currently gives no useful warnings about this
- [ ] I searched through https://github.com/koalaman/shellcheck/issues and didn't find anything related

#### Here's a snippet or screenshot that shows the problem:

```sh

cat /etc/lsb-release 
DISTRIB_ID=Ubuntu
DISTRIB_RELEASE=16.04
DISTRIB_CODENAME=xenial
DISTRIB_DESCRIPTION="Ubuntu 16.04.7 LTS"

```

#### Here's what I wanted or expected to see:

Created a new instance in Google Cloud running 16.04.

```
apt-get install -y shellcheck
Reading package lists... Done
Building dependency tree       
Reading state information... Done
E: Unable to locate package shellcheck
root@instance-1:~# sudo apt-get install -y shellcheck
Reading package lists... Done
Building dependency tree       
Reading state information... Done
E: Unable to locate package shellcheck
```
Do i need to use cabal?

https://kakakakakku.hatenablog.com/?page=1484222260

**Comment by brother** [comment]
Yoiu need to hqave the universe repo active, check that first.
